Src/WindowsServer/WindowsServer.Shared.Tests/EnvironmentVariableMonitorTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.ApplicationInsights.WindowsServer.Implementation;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Assert = Xunit.Assert;
+
+    [TestClass]
+    public class EnvironmentVariableMonitorTests
+    {
+        [TestMethod]
+        public void EnsureInstanceWorksAsIntended()
+        {
+            Assert.NotNull(AppServiceEnvVarMonitor.Instance);
+        }
+
+        [TestMethod]
+        public void EnsureEnvironmentVariablesAreCapturedImmediately()
+        {
+            var envVars = GetCurrentAppServiceEnvironmentVariableValues();
```

Comment:
No cleanup of ENV vars?

Reply:
Thanks for the catch. Oversight on my part (did it in one place, not the other). Updated to use `[TestInitialize]` and `[TestCleanup]` here as well.